packages/flutter/lib/src/material/dropdown_menu.dart

```diff
@@ -44,6 +44,7 @@ class DropdownMenuEntry<T> {
   const DropdownMenuEntry({
     required this.value,
     required this.label,
```

Comment:
Could we improve the `required label` later on? Somebody could now try to specify only `labelWidget`, but then they have to do `label: ''` as well. What if we have only a `labelWidget`, and provide a `.label(String label)` named constructor instead? (This is a breaking change though)

Reply:
I think we always need a label, otherwise the search won't work, since the search always needs a text.